kernel/src/hil/keyboard.rs

```diff
@@ -0,0 +1,25 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2025.
+
+//! Interface for keyboard key presses.
+
+use crate::ErrorCode;
+
+/// Receiver for keyboard key presses.
+pub trait KeyboardClient {
+    /// Called when one or more keys are pressed or un-pressed.
+    ///
+    /// `keys` is an array of `(key_code, is_pressed)` tuples. `key_code` is the
+    /// same as the codes used by Linux to identify different keys.
```

Comment:
Can you put a reference to the list of key codes into the HIL's documentation, as you have in the driver itself?

Reply:
Added